packages/core/strapi/lib/core/loaders/plugins/get-enabled-plugins.js

```diff
@@ -60,7 +60,13 @@ const getEnabledPlugins = async strapi => {
   const installedPlugins = {};
   for (const dep in strapi.config.get('info.dependencies', {})) {
     const packagePath = join(dep, 'package.json');
-    const packageInfo = require(packagePath);
+    let packageInfo;
+    try {
+      packageInfo = require(packagePath);
+    } catch {
+      // Some packages, including firebase-admin, do not export package.json.
```

Comment:
can you remove the firebase-admin example from the comment :)

Reply:
```suggestion
```